CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * Bug Fixes
     * If self.default_to_shell is true, then redirection and piping are now properly passed to the shell. Previously it was truncated.
     * Submenus now call all hooks, it used to just call precmd and postcmd.
+    * Fixed ``AttributeError`` on Windows when running a ``select`` command cause by **pyreadline** not implementing ``remove_history_item``
```

Comment:
This is an important bug fix.  Once this merges please backport all of this code (or at least this part) to the **python2** branch.